### PR TITLE
Fix input-image tests

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | Need `.test.js` in the file name<br>imagem<br>and after fixing the name the tests warned about these problems:<br>imagem<br>imagem |
| Dependencies | -- |
| Decisions | Renamed "input-image.js" to "input-image.test,js" and fixed the component props problems  |
| Screenshot | imagem<br>imagem |

### CODE REVIEW

File: vue/components/ui/molecules/input-image/input-image.vue [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <template>
     <upload-area
         class="input-image"
-        v-bind:files.sync="files"
```
Comment (gcandal) [inline vue/components/ui/molecules/input-image/input-image.vue:4]: Why?

**Comment by NFSS10** [comment]
@gcandal Why did you assigned this back to me? Is something wrong?

**Comment by gcandal** [comment]
Forgot to submit the comment, sorry. It's done now.

File: vue/components/ui/molecules/input-image/input-image.vue (hunk 1 above)
Comment (NFSS10) [reply vue/components/ui/molecules/input-image/input-image.vue:4]: See the Issue section of this PR.
Basically, the tests warned of this dead code. This `sync` does nothing, as isn't referenced anywhere in the code + if you check [this line](https://github.com/ripe-tech/ripe-components-vue/blob/a52cb35274b64dbcf8cba9d2ea9f82c40b7accc7/vue/components/ui/molecules/input-image/input-image.vue#L6 it seems to handle the files update event.

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**